website/docs/terms/etl.md

```diff
@@ -0,0 +1,124 @@
+---
+id: etl
+title: ETL
+displayText: ETL
+hoverSnippet: Extract, Transform, Load (ETL)is the process of first extracting data from a data source, transforming it, and then loading it into a target data warehouse.
+---
+
+ETL, or “Extract, Transform, Load”, is the process of first extracting data from a data source, transforming it, and then loading it into a target data platform. ETL is contrasted with the newer <Term id="elt">Extract, Load, Transform (ELT)</Term> workflow, where transformation occurs after data has been loaded into the target data warehouse.
```

Comment:
I think this is using the old version of the intro. The new one: 


ETL, or “Extract, Transform, Load”, is the process of first extracting data from a data source, transforming it, and then loading it into a target data warehouse. In ETL workflows, much of the meaningful data transformation occurs outside this primary pipeline in a downstream business intelligence (BI) platform.

ETL is contrasted with the newer ELT (Extract, Load, Transform) workflow, where transformation occurs after data has been loaded into the target data warehouse. In many ways, the ETL workflow could have been renamed the ETLT workflow, because a considerable portion of meaningful data transformations happen outside the data pipeline. The same transformations can occur in both ETL and ELT workflows, the primary difference is when (inside or outside the primary ETL workflow) and where the data is transformed (ETL platform/BI tool/data warehouse).

The ETL workflow is still an effective method to create pipelines to bring data from different sources to end business users. However, the past decade has brought an immense amount of innovation in this sphere. There are now a variety of low or no-code data extractors, a code-based and version controlled transformation layer, and several affordable cloud storage options that have allowed ELT to become a competitive analytics workflow.

It’s important to talk about ETL and understand how it works, where it provides value, and how it can hold people back. If we don’t talk about the benefits and drawbacks of systems, how can we expect to improve them?

Reply:
Great catch once again! Fixed